content/docs/command-reference/diff.md

```diff
@@ -19,10 +19,10 @@ positional arguments:
 
 ## Description
 
-Prints a list of files and directories added, modified, deleted in a Git commit
-`b_rev` as compared to another Git commit `a_rev`. Both `a_rev` and `b_rev`
-accept any [Git revision](https://git-scm.com/docs/gitrevisions) - branch or tag
-name, Git commit hash, etc.
+Prints a list of files and directories added, modified, renamed, and deleted in
```

Comment:
thanks! 👍

Reply:
Minor: what about data modified AND renamed? Is it listed as renamed or as modified (or both)? If ambiguous maybe let's change and->or :

```suggestion
Prints a list of files and directories added, modified, renamed, or deleted in
```

Reply:
Only explicit renames (with no data modification) are supported right now. If a file is both renamed and modified, it will show up as a deletion plus an add like in prior DVC versions.